bin/ajax

```diff
@@ -869,6 +865,7 @@ if __name__ == '__main__':
     daq_client = pymongo.MongoClient(f"mongodb://{daq_db_username}:{daq_db_password}@xenon1t-daq:27017/admin")
     daq_db = daq_client[daq_db_name]
     log_coll = daq_db['log']
+    daq_db.command('ping')

```

Comment:
Splitting hairs here, but both databases run on the same server instance, so I don't see a situation where we can ping one but not the other.

Reply:
If we use the wrong credentials for one we will be able to ping the other.

Reply:
This is true, but credential-handling comes with clean error message, and I wouldn't expect it to be a lasting issue.